latex/delegation.tex

```diff
@@ -51,7 +51,7 @@
   \begin{equation*}
     \begin{array}{r@{~\in~}lr}
       a & \AddrRWD & \text{reward address} \\
-      slot & \Slot & \text{slot}\\
+      slot & \Slot & \text{absolute slot}\\
```

Comment:
what is an absolute slot?

Reply:
great question!

Say there were 100 slots per epoch. And pretend for a moment that a block is published every slot.  You could then refer to the slot for the 102nd block either `102` or `(1, 2)` (assuming we start with epoch 0).  In cardano-ls, they following names are used: `102` is the `FlatSlotId`, `(1,2)` is the `SlotId`, and `2` is the `LocalSlotIndex`.

I have others refer to the FlatSlotId as an "absolute slot", and I liked that term.  But I'm open to suggestions! Any way we go, though, we'll need to explain the terminology.

Reply:
ok, this was what I roughly had in mind, thanks for the explication!